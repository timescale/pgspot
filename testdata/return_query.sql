
create function ret_query() returns table(y int)
as $func$
declare
  a text;
  b text;
begin
    return query execute $$select format('abc',1,2)::int$$;
    return query execute $$select format('abc',$1,2)::int$$ using a;
    return query execute b;
    return query execute b || b;
end;
$func$ language plpgsql;

