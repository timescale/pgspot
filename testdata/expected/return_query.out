PS005: Function without explicit search_path: ret_query(y integer) at line 2
PS016: Unqualified function call: format at line 2
PS016: Unqualified function call: format at line 2

 Errors: 0 Warnings: 3 Unknown: 0 

